llama_index/vector_stores/google/generativeai/base.py

```diff
@@ -39,6 +39,32 @@
 """
 
 
+def set_google_auth_credentials(auth_creds: "credentials.Credentials") -> None:
+    try:
+        import llama_index.vector_stores.google.generativeai.genai_extension as genaix
+    except ImportError:
+        raise ImportError(_import_err_msg)
+    genaix.set_config(genaix.Config.with_credentials(auth_creds))
+
+
+"""Sets a Google Auth credentials such as using a service account.
+
+Example:
+    from google.oauth2 import service_account
+    credentials = service_account.Credentials.from_service_account_file(
+        "/path/to/service.json",
+        scopes=[
+            "https://www.googleapis.com/auth/cloud-platform",
+            "https://www.googleapis.com/auth/generative-language.retriever",
+        ],
+    )
+    set_google_auth_credentials(credentials)
+
+Refer to for documentation more information:
+https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount.
+"""
+
+
```

Comment:
I don't think we need this function. We could just use set_config like this:

    set_config(Config(credentials=<whatever>))

Less API footprint this way.

Reply:
Thanks, fixed